spec/features/irs_attempts_api/event_tracking_spec.rb

```diff
@@ -31,10 +31,18 @@
       sign_in_user(user)
 
       events = irs_attempts_api_tracked_events(timestamp: Time.zone.now)
-      expected_event_types = ['email-and-password-auth', 'mfa-verify-phone-otp-sent']
+      expected_event_types = %w[email-and-password-auth mfa-verify-phone-otp-sent]
 
       received_event_types = events.map(&:event_type)
 
+    expect(events.count).to eq received_event_types.count
+    expect(received_event_types).to match_array(expected_event_types)
+
+    metadata = events.first.event_metadata
+    expect(metadata[:user_ip_address]).to eq '127.0.0.1'
+    expect(metadata[:irs_application_url]).to eq 'http://localhost:7654/auth/result'
+    expect(metadata[:unique_session_id]).to be_a(String)
+    expect(metadata[:success]).to be_truthy
       expect(events.count).to be > 0
       expect(received_event_types.sort).to eq(expected_event_types.sort)
```

Comment:
```suggestion
```

38-39 was supposed to replace this but I guess I messed up resolving the conflict.